assets/css/gitshame.css

```diff
@@ -1,9 +1,62 @@
+:root {
+  /*https://color.adobe.com/creamy-grey-color-theme-7775*/
```

Comment:
That's where the colors on the page are coming from. It originally had variables for the colors. Ugh css is awful...I need to find out how to make stylus work.